shared/oval/package_ntp_installed.xml

```diff
@@ -5,10 +5,14 @@
     <metadata>
       <title>Package ntp Installed</title>
       <affected family="unix">
-        <platform>multi_platform_rhel</platform>
+        <platform>CentOS 4</platform>
+        <platform>CentOS 5</platform>
```

Comment:
@shawndwells 
> +CentOS 6

No. This isn't attempting to add support for CentOS 6, just combine the existing ```<platform>``` tags so there won't be unused OVALs when running on RHEL-6 with oscap-1.0.x (but already supported products will still be supported).